### DISCUSSION TITLE
CTKCombobox issue with multiple monitors

### DISCUSSION THREAD

**Discussion by ricleonrod** [TomSchimansky/CustomTkinter#1650] [Q&A]
I have a a flux of screens and I am using the method destroy() from Widget to remove a form and place a new one, the problem is when I am using CTKComboBox in one of those forms and I move the screen to another monitor the resize event is looking for the dropdownmeny from that CTKCombo but, CTK does not have DropdownMenu in CTKCombo then the application is closed.

I had to use a regular tkinter combobox and it worked properly while changing monitors.

**this is the message in the terminal: _tkinter.TclError: invalid command name ".!questions_form.!ctkcombobox.!dropdownmenu"**

**Comment by Akascape** [comment]
@ricleonrod Try destroying the dropdown menu using `combobox._dropdown_menu.destroy()` before destroying the ctkcombobox.

btw, you can also try this new dropdown menu here: 
https://github.com/Akascape/CTkScrollableDropdown

![screenshots](https://user-images.githubusercontent.com/89206401/236677843-8d8b76fd-6145-47b1-8f4d-b6a64b08e1ea.png)
